modules/jaeger-config-storage.adoc

```diff
@@ -189,18 +189,18 @@ spec:
 [id="jaeger-config-external-es_{context}"]
 == Connecting to an existing Elasticsearch instance
 
-Jaeger also allows you to use an existing (self-provisioned) Elasticsearch cluster for storage. You do this by specifying the URL of the existing cluster as the `spec:storage:options:es:server-urls` value in your configuration.
+Jaeger also allows you to use an already existing Elasticsearch cluster for storage (that is, an instance that was not auto-provisioned by the Jaeger Operator calling the Elasticsearch Operator). You do this by specifying the URL of the existing cluster as the `spec:storage:options:es:server-urls` value in your configuration.
```

Comment:
> the Elasticsearch Operator

We should make it clear that it is Red Hat's cluster logging elasticsearch operator and not the upstream elastic.co operator.